src/cascadia/TerminalSettingsEditor/ColorSchemes.cpp

```diff
@@ -71,7 +72,8 @@ namespace winrt::Microsoft::Terminal::Settings::Editor::implementation
         for (uint8_t i = 0; i < TableColorNames.size(); ++i)
         {
             auto entry = winrt::make<ColorTableEntry>(i, Windows::UI::Color{ 0, 0, 0, 0 });
-            _CurrentColorTable.Append(entry);
+            _CurrentNonBrightColorTable.Append(entry);
+            _CurrentBrightColorTable.Append(entry);
```

Comment:
Ah. So here, we're iterating over the `TableColorNames` and adding them to the two color tables right? But the first 8 are non-bright colors, and the remaining ones are bright colors. So, codify that with something like this:
```
if (i < 8)
{
    // add to the non-bright color table
}
else
{
    // add to the bright color table
}
```